src/operator/tensor/elemwise_binary_broadcast_op_basic.cc

```diff
@@ -11,7 +11,23 @@ namespace mxnet {
 namespace op {
 MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(broadcast_add)
 .add_alias("broadcast_plus")
-.describe(R"code(Add arguments, element-wise with broadcasting.
+.describe(R"code(Returns element-wise sum of the input arrays with broadcasting.
+
+Both `broadcast_add` and `broadcast_plus` behave and work the same.
```

Comment:
`broadcast_plus` is an alias to the function `broadcast_add`.
`broadcast_minus` is an alias to the function `broadcast_sub`